### PR TITLE
add openSUSE Leap 42.2 templates

### PR DESCRIPTION
**Pull request by dirkmueller**
Right now a copy of 42.1

### CODE REVIEW

File: openSUSE-42.2/config.kiwi [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="6.1" name="openSUSE-42.2-docker-guest">
+  <description type="system">
+    <author>Flavio Castelli</author>
+    <contact>fcastelli@suse.com</contact>
+    <specification>openSUSE 42.2 docker container</specification>
+  </description>
+  <preferences>
+    <type image="docker" container="os132">
```
Comment (flavio) [inline openSUSE-42.2/config.kiwi]: Can you change `container="os132"` to `container="os422"`?

File: openSUSE-42.2/config.kiwi [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <package name="dracut"/>
+    <package name="elfutils"/>
+    <package name="filesystem"/>
+    <package name="fipscheck"/>
+    <package name="glibc-locale"/>
+    <package name="ncurses-utils"/>
+    <package name="pkg-config"/>
+    <package name="sg3_utils"/>
+  </packages>
+    <packages type="delete">
```
Comment (flavio) [inline openSUSE-42.2/config.kiwi]: Sorry for the nipticking, but can you fix the indentation here?

**Comment by flavio** [comment]
LGTM, sorry for the nitpicking. I realize these are inherited "issues" from the template you copied

**Comment by dirkmueller** [comment]
@flavio done

**Comment by tboerger** [comment]
LGTM

**Pull request merged**